content/blog/analysis-with-RCrunch.Rmd

```diff
@@ -0,0 +1,300 @@
+---
+title: "Analysis with Crunch's R Package"
+description: "Using Crunch to speed up common analysis steps"
+tags: []
+categories: ["general"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: markdown_github
+---
+
+```{r set-options, echo=FALSE}
+options(width = 110)
+```
+
+Crunch provides a collaborative platform for data analytics. Our app allows for simple, visual exploration and analysis, but we know that data scientists and developers want to harness the power of R, Python, and other languages to manipulate, cleanse, and analyze data. Crunch was designed to make collaboration easy and this doesn't stop at the transition between Crunch servers and languages like R and Python. Crunch provides an [R package](https://github.com/crunch-io/rcrunch/) that makes repetitive tasks easy to automate, in a way that feels like an integrated part of R. 
+
+Crunch's R package makes many common cleaning and analysis operations easy. Best of all, it minimizes the amount of data we need to transfer back and forth between Crunch and our local R session. This not only speeds up analysis, but updates data in Crunch instantly so collaborators can see the changes and work with the most up-to-date data as possible.
+
+## Setup
+First we need to load the Crunch package, sign in with our Crunch credentials, and then load the dataset that we would like to analyze. We are using data from [The Stack Overflow annual developer survey](https://insights.stackoverflow.com/survey/?utm_source=so-owned&utm_medium=blog&utm_campaign=dev-survey-2017&utm_content=blog-link&utm_term=data).
```

Comment:
We’ve already imported it, and and we’ll show in another post some import and cleaning steps (I started and may one day finish a post using an r notebook to import from file)